pkg/operator/target_config_reconciler.go

```diff
@@ -324,6 +359,24 @@ func (c TargetConfigReconciler) sync(ctx context.Context, syncCtx factory.SyncCo
 	specAnnotations["service/"+webhookService.Name] = webhookService.GetResourceVersion()
 
 	// From here, we will create our cluster wide resources.
+	err = c.manageAPIService(ownerReference)
```

Comment:
Still need to understand if I need to specify the `specAnnotations` for the assets being added.